### ISSUE TITLE
Topic Frequencies

### ISSUE THREAD

**Issue by SamMckaylin** [Armand1/Evolution-Revolutions#5]
I've managed to get the topics analysis to work and output plots. Comparing to those in "The Nature of Ecology and Evolutionary Biology" frequencies are very different. Do you know how the data was subset in that case? I'm going to change the error bars from standard deviation to standard error + change the plot to a three year moving average but wanted to understand what decision you made with regards to subsetting and why? Currently I am using all of the ecology and evolution papers from 1850-2010.

Plot from current analysis - Y axis made equal to the below plot to demonstrate the differences in frequencies
![Screenshot 2020-04-13 at 13 33 46](https://user-images.githubusercontent.com/61549133/79120711-73814800-7d8b-11ea-93d3-f3f165fa7f1f.png)

Plot from Nature of evolution and ecology
<img width="821" alt="Screenshot 2020-04-09 at 10 57 26" src="https://user-images.githubusercontent.com/61549133/79115426-b4726000-7d7d-11ea-8978-261e43edca25.png">

**Comment by Armand1** [comment]
@SJMcKay --- I don't think this is a subsetting difference.  I am doing to send you two R files --- they're not super neat but they should help you identify why you're not getting my results.

The first, "means and standard deviations for all topics" gets the means and SDs for all topics by year.  It outputs a file called "EEpapers_topics_byyear_05.csv"

The second, "historical topics" takes "EEpapers_topics_byyear_05.csv" as an input and produces my figures above. 

Your job is to look at them and figure out why you're not getting the same thing. Do this ASAP and we'll have a Skype meeting this pm or tomorrow.

**Comment by SJMcKay** [comment]
<img width="821" alt="Screenshot 2020-04-09 at 10 57 26" src="https://user-images.githubusercontent.com/61549133/79369176-72494a00-7f48-11ea-8d8f-68c0278c4f30.png">

**Comment by SamMckaylin** [comment]
![Screenshot 2020-04-13 at 11 14 04](https://user-images.githubusercontent.com/61549133/79369207-7d9c7580-7f48-11ea-9817-124731319708.png)

**Comment by SamMckaylin** [comment]
For reference my original climate plot (using the wrong data) looked like this:
![Screenshot 2020-04-15 at 19 16 10](https://user-images.githubusercontent.com/61549133/79372295-b1c66500-7f4d-11ea-9b4b-c6729c544553.png)

**Comment by Armand1** [comment]
Exactly.  An interesting lesson here in being very careful how you graph things.  Can't be taught --- can only be learned; part of the craft. So well done.
